apps/proposer/config/default.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  SIGNING_KEY:
+    process.env.SIGNING_KEY || '0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69d',
+  PROPOSER_HOST: process.env.PROPOSER_HOST || 'localhost',
+  PROPOSER_PORT: process.env.PROPOSER_PORT || 8092,
+  CLIENT_HOST: process.env.CLIENT_HOST || 'localhost',
```

Comment:
Why do we need a CLIENT?

Reply:
Because `nf3` expects one to be passed to the constructor. Again, we can then tweak the SDK to not need it, but doing so now would cause conflicts in the future. We can add an issue to the SDK repo though